### ISSUE TITLE
Do you plan to release code for training on custom datasets? 

### ISSUE THREAD

**Issue by nishanthcgit** [openai/CLIP#11]
Hi, loving this research and have been trying out zero shot classification a lot. This is amazing work and thanks so much for releasing it in a way we can try it out. 
I wanted to know if openai has any plans of releasing training code etc for us to play around with our own datasets.

**Comment by nishanthcgit** [comment]
Just realized this is a duplicate of https://github.com/openai/CLIP/issues/9

**Comment by jongwook** [comment]
Hello, thanks for your interest!

We don't plan to release training code at this point, but it would look like a standard Pytorch training loop that minimizes the cross entropy loss of the returned logits against the correct labels, i.e. `torch.arange(batch_size)`.
